client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -142,6 +109,9 @@ class HandleEmailedLinkForm extends React.Component {
 		return (
 			<EmptyContent
 				action={ action }
+				className={ classNames( {
+					'magic-login__isfetchingauth': isFetching,
```

Comment:
Thanks! I could have sworn I got a linter warning when I wrote it like that before, but all good now.